pymtl3/datatypes/test/helpers_test.py

```diff
@@ -0,0 +1,82 @@
+"""
+==========================================================================
+helpers_test.py
+==========================================================================
+Test cases for helper functions
+
+Author : Shunning Jiang
+  Date : Nov 30, 2019
+"""
+
+from pymtl3.datatypes import *
+from pymtl3.datatypes.helpers import _get_bitstruct_inst_all_classes
```

Comment:
This helper is used here. If it appears in a test then we might want to expose it instead of prefixing it with `_`